### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by morecallan**
How to test:
- Serve locally, go to index
- Make sure the navigation can bring you to employee page & profile page
- Make sure that you can navigate from those pages as well
- Make sure each page has a different title

### CODE REVIEW

File: html/employees.html [hunk 1]
```diff
@@ -4,12 +4,20 @@
...
+    <a href="./../index.html"><h2 class="logo__navigation">Dino-Mate</h2></a>
+    <ul class="list__navigation">
+      <li><a href="./employees.html">About Us</a></li>
+      <li><a href="./profile.html">Profile</a></li>
+    </ul>
+  </nav>
   <h2 class="about-us-header">About Us</h2>
-  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas sed natus facilis exercitationem quis alias eligendi sunt fugiat deserunt aperiam nulla, eius nisi iure consectetur magnam perferendis itaque molestiae dolores?</p>
+  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas sed natus facilis exercitationem quis alias eligendi
+    sunt fugiat deserunt aperiam nulla, eius nisi iure consectetur magnam perferendis itaque molestiae dolores?</p>
```
Comment (rousell) [inline html/employees.html:21]: can you fix this spacing?

File: index.html [hunk 2]
```diff
@@ -1,13 +1,22 @@
 <!DOCTYPE html>
 <html lang="en">
+
```
Comment (rousell) [inline index.html:3]: space?

File: index.html [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Dinomate</title>
-  <link rel="stylesheet" href="main.css">
+  <link rel="stylesheet" href="./main.css">
+  <title>DinoMate</title>
 </head>
+
```
Comment (rousell) [inline index.html:11]: space?

**Review by rousell — COMMENTED** [review]
some spacing things!

**Comment by rousell** [comment]
Other than that spacing this looks good! Merge away after you fix it

**Comment by morecallan** [comment]
Autoformatting issues

**Pull request merged**